internal/store/cluster_installation_test.go

```diff
@@ -735,7 +735,7 @@ func TestSwitchDNS(t *testing.T) {
 	for _, ci := range clusterInstallations {
 		installationIDs = append(installationIDs, ci.InstallationID)
 	}
-	err = sqlStore.SwitchDNS(oldCIsIDs, newCIsIDs, installationIDs)
+	err = sqlStore.SwitchDNS(oldCIsIDs, newCIsIDs, installationIDs, nil)
```

Comment:
I think it would be good to add at least one hibernating installation to verify it is switched to the correct state.